### PR TITLE
Typo in en.json

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
#### Summary
Fixing some more typo's in en.json

#### Related Pull Requests
Not all typo's got fixed in https://github.com/mattermost/mattermost-webapp/pull/9856
@amyblais and @JulienTant 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -2418,7 +2418,7 @@
   "announcement_bar.error.past_grace": "Enterprise license is expired and some features may be disabled. Please contact your System Administrator for details.",
   "announcement_bar.error.preview_mode": "Preview Mode: Email notifications have not been configured.",
   "announcement_bar.error.purchase_a_license_now": "Purchase a License Now",
-  "announcement_bar.error.site_url_gitlab.full": "Please configure your [site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) either on the [System Console](/admin_console/environment/web_server) or, if you're using GitLab Mattermost, in gitlab.rb.",
+  "announcement_bar.error.site_url_gitlab.full": "Please configure your [site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) either on the [System Console](/admin_console/environment/web_server) or, if you're using GitLab Mattermost, in Gitlab.rb.",
```
Comment (JulienTant) [inline i18n/en.json:2421]: Could you make sure that this is not an error? I don't believe the filename contains the capital `G`

**Review by JulienTant — CHANGES_REQUESTED** [review]
Thanks, for porting my comment from the previous PR here

File: i18n/en.json (hunk 1 above)
Comment (ctlaltdieliet) [reply i18n/en.json]: I fixed that!

**Review by JulienTant — APPROVED** [review]
LGTM thank you!

**Review by cwarnermm — APPROVED** [review]
Thank you!

**Review by hmhealey — APPROVED** [review]

**Review by hmhealey — CHANGES_REQUESTED** [review]
Thanks for the PR! Could you also update the default strings that appear in the code for each of those? Like the first one also shows up [here](https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/billing/plan_details/plan_details.tsx#L285)

**Comment by cwarnermm** [comment]
@ctlaltdieliet - Let me know if you need any help with the follow-on ask. Happy to help!

**Comment by ctlaltdieliet** [comment]
I think I did manage to do this @cwarnermm and @hmhealey

**Pull request merged**